library.properties

```
@@ -1,8 +1,8 @@
 name=Arduino_LSM6DS3
-version=1.0.0
+version=1.1.0
```

```suggestion
version=1.0.0
```
I think it's best to leave the versioning to the repository maintainers. The version is normally bumped right before the release, so it might cause confusion to have it pre-bumped.